src/storage/statistics_sr.c

```diff
@@ -392,9 +394,22 @@ xstats_update_all_statistics (THREAD_ENTRY * thread_p, bool with_fullscan)
   while (heap_next (thread_p, &root_hfid, oid_Root_class_oid, &class_oid, &recdes, &scan_cache, COPY) == S_SUCCESS)
     {
 #if !defined(NDEBUG)
-      classname = or_class_name (&recdes);
+      string = NULL;
+      alloced_string = 0;
+      error = or_class_name (&recdes, &string, &alloced_string);
+      if (error != NO_ERROR)
+	{
+	  ASSERT_ERROR ();
+	  goto exit_on_error;
+	}
+      classname = string;
       assert (classname != NULL);
-      assert (strlen (classname) < 255);
+      assert (strlen (classname) < 255);	// to be: DB_MAX_FULL_CLASS_LENGTH
```

Comment:
A macro definition looks better to hard-coded 255.

Reply:
Does that mean it's better to change it to a macro variable?
Later this will be changed to a macro variable by adding DB_MAX_FULL_CLASS_LENGTH.
For now, I'll set it to the DB_MAX_IDENTIFIER_LENGTH macro variable.